src/main/kotlin/racingcar/domain/CarName.kt

```diff
@@ -0,0 +1,20 @@
+package racingcar.domain
+
+class CarName(name: String) {
+    val value: String
+
+    init {
+        if (name.length > 5) {
+            throw IllegalArgumentException(ERROR_MESSAGE)
+        }
+        this.value = name
+    }
+
+    override fun toString(): String {
+        return value
+    }
```

Comment:
```suggestion
class CarName(val name: String) {

    init {
        require(name.length > 5) { ERROR_MESSAGE }
    }
```

name을 property로 사용하면 더 자연스럽지 않을까요?